typescript/configuration.md

```diff
@@ -207,6 +207,6 @@ could be trivial errors becoming runtime errors.
 ### Using the "types" property
 
 The `"types"` property in `"compilerOptions"` can be used to specify arbitrary
-type definitions to include when type checking a programme. For more information
```

Comment:
This is actually correct if we're using British-English, but to keep it consistent let's change it as proposed.